components/charts/__tests__/ChartCustomizations.test.tsx

```
@@ -22,292 +21,149 @@ describe('ChartCustomizations', () => {
-      expect(screen.getByLabelText('Show Legend')).toBeInTheDocument();
-
-      await user.click(screen.getByLabelText('Show Tooltip on Hover'));
-      expect(mockOnChange).toHaveBeenCalled();
-    });
-
-    it('should show legend customization when legend is visible', () => {
-      const formData = createFormData('bar', { customizations: { showLegend: true } });
-      render(<ChartCustomizations chartType="bar" formData={formData} onChange={mockOnChange} />);
-
-      expect(screen.getByText('Legend Display')).toBeInTheDocument();
+      screen.getAllByRole('switch').forEach((s) => expect(s).toBeDisabled());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)